katalog/dual-nginx/default-ingress.yml

```diff
@@ -4,18 +4,10 @@ metadata:
   name: default-internal
   annotations:
     kubernetes.io/ingress.class: internal
-spec:
```

Comment:
are these ingress still needed? if it is the case, the `spec` section cannot be nil, otherwise please remove them

Reply:
That's a good question, I'm not really sure. I believe we are using them as a sort of "catch-all", but I'm no sure we need them. What happens if there are no ingress defined?

Reply:
if no ingress is defined and you reach nginx ingress controller you received a 404 error, which should be the desired behaviour

Reply:
LGTM. Let's drop them